components/jest.config.js

```diff
@@ -1,7 +1,4 @@
 module.exports = {
   preset: '@vue/cli-plugin-unit-jest/presets/typescript-and-babel',
-  transform: {
-    '^.+\\.vue$': 'vue-jest',
-  },
```

Comment:
`vue-jest` deprecated. `vue3-jest` is installed
https://github.com/vuejs/vue-jest#installation